packages/network-core/test/basic.test.ts

```diff
@@ -3,6 +3,8 @@ import * as nock from 'nock';
 import fetch from 'node-fetch';
 import { Network, NetworkClient, Response, setUrl } from '../src';
 
+console.log(nock('aaa'));
```

Comment:
TODO: check why tests throw an error `TypeError: nock is not a function`. It causes this test suit to fail